x/bsonx/bsoncore/value.go

```diff
@@ -978,38 +978,3 @@ func sortStringAlphebeticAscending(s string) string {
 	sort.Sort(ss)
 	return string([]rune(ss))
 }
-
-func docAsArray(d Document, debug bool) string {
```

Comment:
Nice, removing this in favor of `Array.DebugString` / `Array.String` was a good call.